French/Text/AbilityText.xml

```diff
@@ -216,7 +216,7 @@
     </StringTable>
     <StringTable>
         <Label>StarfaringAbility_Dec</Label>
-        <String><![CDATA[+25% de Portée de Navire. Commence avec la capacité de détecter et voyager sur les Courants de Sous-espace.]]></String>
+        <String><![CDATA[+25% d'autonomie pour les navires. Commence avec l'abilité de detecter et d'utiliser les flux subspatiaux.]]></String>

```

Comment:
I wonder though if "Portée" will be used everywhere else in the files though instead of "Autonomie"; Portée is still adequate for this as it refer to your maximum range, while Autonomie might relate more to how many moves you can make between refueling. I kind of prefer Portée for that one.

Flux subspatiaux par contre ça c'est vraiment mieux!! j'adopte.

Reply:
OK make sense